conans/client/command.py

```diff
@@ -2045,7 +2045,7 @@ def run(self, *args):
                 raise ConanException("Unknown command %s" % str(exc))
 
             if is_config_install_scheduled(self._conan) and \
-               (command != "config" or (command == "config" and args[0] != "install")):
+               (command != "config" or (command == "config" and args[0][1] != "install")):
```

Comment:
args now is tuple with a list ([]) that's why config install triggered the scheduler ...